web/src/views/developers/ApiDocsUsersView.vue

```diff
@@ -0,0 +1,111 @@
+<script setup lang="ts">
+	import DocumentationPage from '../../components/DocumentationPage.vue';
+	import ApiDocsSidebar from '../../components/ApiDocsSidebar.vue';
+	import exampleUserResponse from './exampleUserResponse.json?raw';
+
+	// @ts-ignore
+	import Prism from 'vue-prism-component';
+	import 'prismjs/components/prism-json';
+	import 'prismjs/components/prism-http';
+
+	const baseUrl = import.meta.env.VITE_API_URL;
+</script>
+
+<template>
+	<DocumentationPage title="User Endpoints">
+		<template #sidebar>
+			<ApiDocsSidebar />
+		</template>
+
+		<p class="mb-4">
+			Updating a user requires that user being currently authenticated via
+			JWT, however, for the time being, that functionality will not be
+			documented as it serves no good purpose for integrators of the API.
+		</p>
+
+		<section class="endpoint">
+			<h2 class="heading">
+				<span class="method">GET</span>
+				/v1/users/:id
+			</h2>
+			<b>Example request (HTTP):</b>
+			<Prism language="http" class="rounded-lg">
+				GET {{ baseUrl }}/v1/users/853034340056039424
+			</Prism>
+
+			<b>Example Response (JSON):</b>
+			<Prism language="json" class="rounded-lg">{{
+				exampleUserResponse
+			}}</Prism>
+		</section>
```

Comment:
Maybe this entire block can be implemented as an element to shorten the lines required to write it out and make the page source simpler to read/develop, e.g. something along the lines of this perhaps:

```jsx
<APIRoute method="GET" path="/v1/users/:id">
    <APIRequest>GET {{baseUrl}}/v1/users/853034340056039424</APIRequest>
    <APIResponse type="json">
        {{ exampleUserResponse }}
    </APIResponse>
</APIRoute>
```

Probably not this exact syntax or layout, but it saves a few lines and repeated code.

Reply:
Not a bad idea, i also forgot to add a link to docs so i'll implement both of these in a minute or two

Reply:
Added the link on the footer. Went ahead and resolved since the current markup is good enough(tm) and I kind of want to just merge and go sleep. I still want to abstract that into a component but it isn't my highest priority right now.